build-scripts/build.sh

```diff
@@ -4,5 +4,6 @@ set -e
 npm install --force
 python build-scripts/licensechecker/licensechecker.py
 npm run test
+#npm run validate:themes -- default # Validates themes
```

Comment:
Are we not running this as part of the build process yet?

Reply:
Also the script is `validate:theme` - not plural

Reply:
Since the theme name changes with every project we'd have to do something to loop through the directory.
It's a good idea. Would make it more immediately usable if you could just validate all themes.

Potentially outside the original scope of this ticket though? What do you think?

Reply:
If it's an easy addition then I'd say it would be worth it - if it's at all tricky then discuss as a separate piece.